src/main/java/io/github/externschool/planner/entity/profile/Person.java

```diff
@@ -29,6 +31,9 @@
     @JoinColumn(name = "id")
     private User user;
```

Comment:
since we move user-person connection to verificationKey entity we don't need this field anymore